features/edit_questions.feature

```diff
@@ -0,0 +1,116 @@
+    And I have a Response Set with the name "Gender Partial"
+    And I have a Response Type with the name "Response Set"
+    And I am logged in as test_author@gmail.com
+    When I go to the dashboard
+    And I click on the menu link for the Question with the content "What is your gender?"
+    And I click on the option to Revise the Question with the content "What is your gender?"
+    And I fill in the "Question" field with "What is your favorite color?"
+    And I fill in the "Description" field with "This is a revised description"
+    And I drag the "Gender Partial" option to the "Selected Response Sets" list
+    And I select the "Response Set" option in the "Response Type" list
+    And I click on the "Add Row" link
+    And I fill in the "value_1" field with "Test Concept 2"
+    And I click on the "remove_0" link
+    And I click on the "Save" button
+    And I should see "What is your favorite color?"
+    And I should see "This is a revised description"
+    And I should not see "New Concept Name"
+    And I should see "Test Concept 2"
+
+  Scenario: Extend Question
+    Given I have a Question with the content "What is your gender?" and the description "This is a question" and the type "MC" and the concept "New Concept Name"
+    And I am logged in as test_author@gmail.com
+    When I go to the list of Questions
+    When I click on the menu link for the Question with the content "What is your gender?"
+    And I click on the option to Details the Question with the content "What is your gender?"
+    And I click on the "Publish" button
+    And I click on the "Extend" button
+    And I fill in the "Description" field with "This is an extended description"
+    And I click on the "Save" button
+    Then I should see "Version: 1"
+    And I should see "Extended from: What is your gender?"
+    And I should see "New Concept"
+    And I should see "This is an extended description"
+    And I should not see "This is a question"
+
+  Scenario: Create New Question from List, test rs add button, and Create a Response Set using New Response Set Modal
+    Given I have a Response Set with the name "Gender Full"
+    And I am logged in as test_author@gmail.com
+    When I go to the dashboard
+    And I click on the create "Questions" dropdown item
+    And I fill in the "Question" field with "What is your favorite color?"
+    And I fill in the "Description" field with "This is a description"
+    And I click on the "select-Gender Full" link
+    Then I click on the "Add New Response Set" button
+    Then I fill in the "response_set_name" field with "New Response Set"
+    And I click on the "Add Response Set" button
+    Then I should see "New Response Set"
+    And I click on the "Save" button
+    And I should see "What is your favorite color?"
+    And I should see "New Response Set"
+
+  Scenario: Test rs add button doesn't allow you to add multiple of the same causing errors
```

Comment:
FYSA, this is the only new test, the rest were just moved to match the form feature changes recently (reducing length of single file for all question related testing) as it was getting long.